draft-wang-ppm-differential-privacy.md

```diff
@@ -536,7 +559,8 @@ class DpMechanism:
         """
         Debias the data due to the added noise, based on the number of
         measurements `meas_count`. This doesn't apply to all DP
-        mechanisms. Some Client-DP mechanisms need this functionality.
+        mechanisms. Some Client randomization mechanisms need this
```

Comment:
From Kunal: we should change Client-DP to Client randomization, and Aggregator-DP to Aggregator randomization. Client or Aggregator-DP typically means DP guarantee, e.g. `EPSILON`-DP, or `(EPSILON, DELTA)`-DP.

Reply:
I agree with the rename. However, do we want some text to explain that the goal of DpPolicy is to achieve Aggregator-DP guarantee, using either client-randomisation or aggregator randomisation.

Reply:
New terminology sounds good to me, just make sure it's consistent.
@wangshan I don't think saying "Aggregator-DP guarantee" is meaningful unless we define it. For now, pure-DP and approximate-DP are the only well-defined terms.

Reply:
nit: I think papers have been referring the guarantee achieved by DP policy to be "aggregate differential privacy". The guarantee achieved at Client is referred to as Client- or local-DP guarantee.

Not a huge blocker: I think we need a section to formally define Client- (or local-) or aggregate- (or central-) DP guarantee, which are currently in {{levels}}. For the pure-DP and approximate-DP we've defined, we've been assuming aggregate DP guarantee.